### DISCUSSION TITLE
[ERROR] Could not resolve "os" (also fs, path, http etc) when building Shopify Remix App

### DISCUSSION THREAD

**Discussion by mbucher100** [remix-run/remix#9555] [Q&A]
I can't get it solved. Does anyone know a solution?

The problem is related to the logger winston & winston-loki and the build process.

![image](https://github.com/remix-run/remix/assets/58312460/8869b5a5-f5e8-4bd6-932e-19fc95881a9f)

Am grateful for every hint.

**Comment by Erij-Maherzia-BEN-BRAHIM** [comment]
have a look at this 👇🏻 it may help you
https://github.com/evanw/esbuild/issues/2409#issuecomment-1198446562
